src/main/kotlin/com/cognifide/gradle/aem/AemPlugin.kt

```diff
@@ -40,8 +41,8 @@ class AemPlugin : Plugin<Project> {
         setupDependentPlugins(project)
         setupExtensions(project)
         setupTasks(project)
-        setupConfigs(project)
-        setupValidation(project)
+        setupConfigurations(project)
```

Comment:
yep configuration in gradle nomenclature is a set of files
but in case of AEM plugin we have just 'config' which means set of properties. I have no idea how to not make it more confusing